daily-challenges/2024-10/07/hunghoang3110.spec.ts

```
@@ -0,0 +1,14 @@
+// Javascript
+function findDuplicateIDs(arrayID) {
+    let arrayIDNotDup = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'arrayIDNotDup' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)